### ISSUE TITLE
Embedded styles ignored

### ISSUE THREAD

**Issue by denvist** [meyfa/php-svg#32]
Style element not present in rendered SVG image
```
<defs
     id="some_id">
    <style
       type="text/css"
       id="some_other_id">
              @import 'http://fonts.fontstorage.com/import/merriweather.css';
              text {font-family: 'Merriweather';}
              .....
    </style>
</defs>
```

**Comment by denvist** [comment]
[result](http://xn--80aafggbkhlniwp7j.online/php-svg/svg.php)
[source](http://xn--80aafggbkhlniwp7j.online/php-svg/body.svg)

library - last version

**Comment by meyfa** [comment]
I added support for the `<defs>` element in 5dc6b2c, which should fix this issue.

**Comment by meyfa** [comment]
Fix contained in release v0.6.1
